compiler/src/IelePrint.hs

```diff
@@ -162,6 +162,13 @@ prettyIeleInst (CallOp (STATICCALLDYN _ _) results allArgs) = case results of
              <> char '(' <> commaList args <> char ')'
              <+> text "gaslimit" <+> gas
          _ -> error "external staticcall instruction must encode at least target and gaslimit arguments"
+prettyIeleInst (CallOp (CALLADDRESS name) results allArgs) = case results of
+  [] -> error "external call instruction must have at least one result"
+  _ -> case allArgs of
+         (acct:[]) ->
+           prettyResults results <+> text "calladdress" <+> prettyName name
+             <+> text "at" <+> acct
+         _ -> error "external staticcall instruction must encode at least target argument"
```

Comment:
This message says "staticcall". This should also expect exactly one result, right?